test/integration/master/synthetic_master_test.go

```diff
@@ -300,7 +300,7 @@ func TestObjectSizeResponses(t *testing.T) {
 
 	client := clientsetv1.NewForConfigOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Groups[api.GroupName].GroupVersion()}})
 
-	const DeploymentMegabyteSize = 100000
+	const DeploymentMegabyteSize = 130000
```

Comment:
Why changing this value?

Reply:
Maybe add another test (or check in the actual test) which verifies this number, i.e. that the body has the right size?

Reply:
For `constructBody`, I think you could pad zero to the number when printing to make it easier to calculate the size of created object, like:

```
 for i := 0; i < size; i++ { 
 	key := val + fmt.Sprintf("%010d", i) // len(key) = len(val) + 10
 	labelsMap[key] = val 
 } 
```